Client/Nimbus/NimbusFeatureFlagLayer.swift

```diff
@@ -83,6 +84,17 @@ final class NimbusFeatureFlagLayer {
         }
     }
 
+    public func checkNimbusConfigForOnboardingNotificationCard(using nimbus: FxNimbus = FxNimbus.shared) -> OnboardingNotificationCardPosition {
```

Comment:
nit: I am bothered by how long this is. Is it inour 120 chars thing? I really need to get around to fixing that rule in Swiftlint. TOO MANY VIOLATIONS! lols